### PR TITLE
add a changelog generator for markdown files and the rtfd

### PR DESCRIPTION
**Pull request by warlof**

### CODE REVIEW

File: development/make-md-changelogs.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+# Warlof Tutsimo
```
Comment (leonjza) [inline development/make-md-changelogs.sh]: Can remove these.

File: development/make-md-changelogs.sh [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+# loic.leuilliot@gmail.com
+
+# The list of packages for which we want to generate markdown formatted changelog
+packages=( api console eveapi notifications services web )
+# The root repository location
+repository_directory="E:/Users/elfaus/PhpstormProjects/seatng/"
```
Comment (leonjza) [inline development/make-md-changelogs.sh]: We should not be having hard coded paths.

Comment (warlof) [inline development/make-md-changelogs.sh]: replaced with `pwd` and add parameters support for both packages list and output directory :)

File: development/make-md-changelogs.sh (hunk 1 above)
Comment (warlof) [inline development/make-md-changelogs.sh]: done

**Pull request merged**